README.md

```diff
@@ -2,12 +2,75 @@
 
 Open-source 4G EnodeB and User emulators developed by [Software Radio Systems (SRS)](https://www.srslte.com/).
 
-## Usage
+## Installation and configuration
 
 ```bash
 juju deploy charmed-osm-srs-enb-ue --channel edge
 ```
 
+The easiest method is to configure and deploy at the same time. For it create a YAML configuration file:
+
+```yaml
+---
+srs-enb-ue:
+  bind-address-subnet: <ipv4 address>
+  enb-name: <enb name>
+  enb-mcc: <mcc>
+  enb-mnc: <mnc default>
+  enb-rf-device-name: <zmq>
+  enb-rf-device-args: <RF Device Name.>
+  ue-usim-algo: <The authentication algorithm to use (MILENAGE or XOR).>
+  ue-nas-apn: <NAS Access Point Name (APN).>
+  ue-device-name: <UE Device Name.>
+  ue-device-args: <UE Device arguments.>
+```
+
+And run the deploy command:
+
+```bash
+juju deploy charmed-osm-srs-enb-ue --config <yaml config file>
+```
+
```

Comment:
All those lines are not necessary in my opinion (33-47). Here we don't want to tell the user how to use charms/Juju.